anoma/namada-interface
# #10 Unit tests for actions and configure in CI

# What does it do
* Adds some unit tests for actions in shielded account, to support refactoring. 
* configures the unit tests to run in CI

# How to test
* run them locally with
```
yarn test:coverage
```
* or observe the test execution in CI in GitHub actions
